src/fmu/ensemble/ensemble.py

```diff
@@ -1416,7 +1416,7 @@ def get_eclgrid(self, props, report=0, agg="mean", active_only=False):
         dframe = grid_index.reset_index().join(corners).join(centre)
         dframe["realizations_active"] = self.global_active.numpy_copy()
         for prop in props:
-            print("Reading the grid property: " + prop)
+            logger.info(f"Reading the grid property: {prop}")
```

Comment:
Isn't flake or pylint complaining that lazy logging is not used here?

Reply:
Fixed :+1: